cowprotocol/explorer
# UI improvements for Release 2.13.0

# Summary

Closes #217
![image](https://user-images.githubusercontent.com/622217/190528604-8ea10752-2347-4bab-a298-b0b3fd677d8f.png)

## To test:
1. Check the main menu alignment.
2. Check the font-family on the AppData page buttons. 
3. Check the arrow animation on the dropdown option on the main menu. It should rotate 180º in the same axis. [Reported here](https://github.com/cowprotocol/explorer/pull/214#issuecomment-1247842006) by @alfetopito 

## NOTE:
We need to solve a major issue regarding [Inter var](https://rsms.me/inter/) font that it's missing. We'll be working in a different PR to solve that.
